README.md

```diff
@@ -1,4 +1,4 @@
- ## Conference Website 
+ ## To Do Web Applicatiom

```

Comment:
[OPTIONAL] Kindly make sure that you have the MIT license for your project. When I tried to open the link in Readme it shows me a broken link.

Reply:
Thanks, @Rustamxon7. I will implement what you have mentioned above.